conjur/argument_parser/_hostfactory_parser.py

```diff
@@ -54,13 +57,16 @@ def _create_hostfactory_parser(self):
 
     @staticmethod
     def _add_hostfactory_create(hostfactory_subparser: ArgparseWrapper):
+        #Needs TW review
         hostfactory_create_name = 'create - Create token for creating hosts ' \
```

Comment:
```suggestion
        hostfactory_create_name = 'create - Create Host Factory token for creating hosts ' \
```

Reply:
Maybe "Generate Host Factory token... " otherwise "create "and "create" is too much "create"